packages/request-handler/src/notifications.ts

```diff
@@ -0,0 +1,20 @@
+import dayjs from "dayjs";
+
+import type { IntegrationKindByCategory } from "@homarr/definitions";
+import type { Notification } from "@homarr/integrations";
+import { createIntegrationAsync } from "@homarr/integrations";
+
+import { createCachedIntegrationRequestHandler } from "./lib/cached-integration-request-handler";
+
+export const notificationsRequestHandler = createCachedIntegrationRequestHandler<
+  Notification[],
+  IntegrationKindByCategory<"notifications">,
+  { topics: string[] }
+>({
+  async requestAsync(integration, input) {
+    const integrationInstance = await createIntegrationAsync(integration);
+    return await integrationInstance.getNotificationsAsync(input.topics);
+  },
+  cacheDuration: dayjs.duration(5, "seconds"),
```

Comment:
You set the interval to 30 seconds but the cache to 5 seconds? Must it really expire so quickly? I'd say the cache could expire after 1 minute or 30 seconds